foursight_core/app_utils.py

```diff
@@ -29,6 +41,8 @@
 logging.basicConfig()
 logger = logging.getLogger(__name__)
 
+version = pkg_resources.get_distribution('foursight-cgap').version
```

Comment:
You might need to retrieve this through some other mechanism (`foursight-cgap` is not always the package, it could be `foursight`).

Reply:
Oh, hm, enough to check for one, and then the other? Or, what other mechanism?

Reply:
Took care of this, using Kent's suggest, below.